docs/UserGuide.md

```diff
@@ -6,10 +6,31 @@
 
 # WedLog User Guide
 
-
 <!-- * Table of Contents -->
 <page-nav-print />
 
+WedLog is a desktop app for wedding planning, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). WedLog allows you to view, categorise and edit information about the people involved in your wedding (e.g. vendors, guests). With WedLog, you can easily track multiple streams of person-based information – RSVP status, dietary requirements, and more.
+
+- [Quick Start](#quick-start)
+- [Features](#features)
+  - [Viewing help: `help`](#viewing-help-help)
+  - [Adding a guest: `guest add`](#adding-a-guest-guest-add)
+  - [Adding a vendor: `vendor add`](#adding-a-vendor-vendor-add)
+  - [Deleting a guest: `guest delete`](#deleting-a-guest-guest-delete)
+  - [Deleting a vendor: `vendor delete`](#deleting-a-vendor-vendor-delete)
+  - [Viewing all guests: `guest list`](#viewing-all-guests-guest-list)
+  - [Viewing all vendors: `vendor list`](#viewing-all-vendors-vendor-list)
+  - [Filtering guests: `guest filter`](#filtering-guests-guest-filter)
+  - [Filtering vendor: `vendor filter`](#filtering-vendors-vendor-filter)
+  - [Undoing last action: `undo`](#undoing-last-action-undo)
+  - [Redoing last action: `redo`](#redoing-last-action-redo)
+  - [Exiting the program: `exit`](#exiting-the-program-exit)
+- [FAQ](#faq)
+- [Known issues](#known-issues)
+- [Command Summary](#command-summary)
+- [Appendix A](#appendix-a-acceptable-values-for-parameters)
```

Comment:
i think it will be helpful to include the content of the appendix (eg Appendix B: Miscellaneous error messages) what do you think?